packages/dd-trace/src/config.js

```diff
@@ -5,10 +5,8 @@ const pkg = require('./pkg')
 const coalesce = require('koalas')
 const scopes = require('../../../ext/scopes')
 const tagger = require('./tagger')
-const id = require('./id')
 const { isTrue, isFalse } = require('./util')
-
-const runtimeId = `${id().toString()}${id().toString()}`
+const uuid = require('./uuid')
```

Comment:
What was wrong with the one-liner to justify the added complexity of an actual UUID implementation?

Reply:
It doesn't produce correctly formatted uuids.

Reply:
Can you clarify? Do you mean the dashes? If that's the case I don't think it makes sense to add so much code just because of that when all that is needed is a `replace`.

Reply:
To be clear, that code is a literal copy/paste of the `crypto.randomUUID` code in Node.js core. It would only exist for as long as that function is not available in all supported versions. IMO it's better to use something with a path to removal than to perpetually use some fake uuid generation code. Also uuids are more than just a random string of hex characters with dashes. It has certain characters that need to have certain values to indicate how they should be parsed and things like that. It's better to have a _real_ uuid function than to potentially blow something up that might try to read the `runtime-id` thinking it should be a valid uuid.

Reply:
Then why not add a dependency on a UUID module? Especially if it will be temporary, then we can just remove the dependency.

Reply:
I went for this approach for a more exact polyfill. Could turn it into a separate module if that would be preferred. Certainly others could benefit for a polyfill. Also probably worth backporting the stuff in Node.js core to v14. 🤔

Reply:
I would definitely prefer this approach, especially for something as common as UUID.

Reply:
Do you prefer to do it in 2 steps? Not gonna block the PR based on this file.

Reply:
I made https://github.com/Qard/crypto-randomuuid and depended on that. The `crypto.randomUUID` method was introduced in 15.x and backported to 14.x, so it shouldn't be too terribly long until we can remove this dependency.